src/pages/Profile.js

```diff
@@ -1,17 +1,74 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Header from '../components/Header';
 import Footer from '../components/Footer';
 
-function Profile() {
+function Profile({ history: { push } }) {
+  function getUserEmail() {
+    const email = JSON.parse(localStorage.getItem('user')) || {};
+    return email.email;
```

Comment:
Não seria melhor chamar a variável `email` de `user`?
```suggestion
    const user = JSON.parse(localStorage.getItem('user')) || {};
    return user.email;
```

Reply:
Também tem como usar a desestruturação se preferir 😉 
```suggestion
    const { email } = JSON.parse(localStorage.getItem('user')) || {};
    return email;
```

Reply:
Legal desestruturar então.